curriculum/http/schedule.md

```diff
@@ -2,53 +2,61 @@
 
 ## Day 1
 
-| Time    | Activity                              | Learning outcomes    |
-| ------- | ------------------------------------- | -------------------- |
-| 09:45   | Check-in                              |                      |
-| 10:00   | [Intro presentation][http-slides]     |                      |
-| 10:30   | [HTTP intro][http-intro]              | HTTP                 |
-| 11:00   | [First-class functions][fc-fns]       | Functions, callbacks |
-| _13:00_ | _Lunch_                               |                      |
-| 14:00   | [Fetch & Promises intro][fetch-intro] | Fetch, Promises      |
-| 16:00   | Tech for Better                       |                      |
+| Time    | Activity                                           | Learning outcomes    |
+| ------- | -------------------------------------------------- | -------------------- |
+| 09:45   | Mentors intro                                      |                      |
+| 09:50   | Check-in                                           |                      |
+| 10:00   | Feedback survey                                    |                      |
+| 10:05   | [Intro presentation][http-slides-25]               | What is HTTP         |
+| 10:30   | HTTP game                                          | How does HTTP work?  |
```

Comment:
I think we said we'd remove this - would we like to remove from schedule?

Reply:
We're going to try and make HTTP game work